WcaOnRails/app/views/polls/edit.html.erb

```diff
@@ -0,0 +1,68 @@
+<% provide(:title, 'Editing poll #' + @poll.id.to_s) %>
+
+<div class="container">
+
+  <h3><%= yield(:title) %></h3>
+
+  <% is_actually_confirmed = Poll.find(@poll.id).confirmed? %>
+  <% if is_actually_confirmed %>
+    <div class="alert alert-danger">
+      This poll is open to voting. Editing question or options is not allowed.
+    </div>
+    <script>
+    $(document).ready(function() {
```

Comment:
I think it would be better to do this without javascript, by passing `disabled: is_actually_confirmed` to each input.